src/server.rs

```diff
@@ -104,8 +104,9 @@ impl Server {
 
     pub fn run(&mut self) -> Result<(), IouError> {
         self.accept()?;
+        let mut wait_count = 1;
         loop {
-            self.ring.submitter().submit_and_wait(1)?;
+            self.ring.submitter().submit_and_wait(wait_count)?;
```

Comment:
I'm not sure we actually want to wait for everything to finish. Wouldn't this mean that if we have multiple sockets, we'll only return from this when all of them are ready?